build.gradle

```diff
@@ -162,12 +167,9 @@ subprojects { subproject ->
 		testImplementation "org.hamcrest:hamcrest-core:$hamcrestVersion"
```

Comment:
For the actual dependencies, should we specify the versions in dependencyManagement section and then just specify group/artifiact in the actual dependencies?

Reply:
may not have to - i followed what SK does.